src/source/Rtp/RtpPacket.c

```diff
@@ -135,7 +135,7 @@ STATUS constructRetransmitRtpPacketFromBytes(PBYTE rawPacket, UINT32 packetLengt
     pPayload = (PBYTE) MEMALLOC(pRtpPacket->payloadLength + SIZEOF(UINT16));
     CHK(pPayload != NULL, STATUS_NOT_ENOUGH_MEMORY);
     // Retransmission payload header is OSN original sequence number
-    putInt16((PINT16) pPayload, pRtpPacket->header.sequenceNumber);
+    putUnalignedInt16BigEndian(pPayload, pRtpPacket->header.sequenceNumber);
```

Comment:
I believe the Endianness library has already been initialized so this should evaluate to the same code. However, this is a GOOD fix as it will ensure no unaligned memory access.